treeherder/model/tasks.py

```diff
@@ -112,7 +112,7 @@ def publish_resultset_action(project, action, resultset_id, requester, times=1):
 
 @task(name='publish-resultset-runnable-job-action')
 def publish_resultset_runnable_job_action(project, resultset_id, requester,
-                                          buildernames):
+                                          buildernames, taskID):
```

Comment:
I think it'd be better to call it ``decisionTaskID`` to make it clearer what it is.  Here and in the other places it's referenced.